skimage/feature/tests/test_basic_features.py

```diff
@@ -1,30 +1,70 @@
 import pytest
 import numpy as np
+
+from skimage._shared.testing import expected_warnings
 from skimage.feature import multiscale_basic_features
 
 
+
 @pytest.mark.parametrize('edges', (False, True))
 @pytest.mark.parametrize('texture', (False, True))
 def test_multiscale_basic_features(edges, texture):
     img = np.zeros((20, 20, 3))
     img[:10] = 1
     img += 0.05 * np.random.randn(*img.shape)
-    features = multiscale_basic_features(img, edges=edges, texture=texture, multichannel=True)
+    with expected_warnings(["'multichannel' is a deprecated argument"]):
+        features = multiscale_basic_features(img, edges=edges, texture=texture,
+                                             multichannel=True)
```

Comment:
Should this not be `channel_axis=-1`, since you test the deprecation below?

Reply:
yes, thanks. I was separating the deprecated tests out more consistently as I went along and tried to come back and do the same here, but missed this one!